### PR TITLE
feat: make db version configurable. Adds some required fields to the module

### PR DESCRIPTION
**Pull request by bmonkman**
## Description

Please explain the changes you made here and link to any relevant issues.

### Checklist

- [ ] Validation tests are passing
- [ ] README.md has been updated after any changes to variables and outputs. See https://github.com/commitdev/terraform-aws-zero/#doc-generation

### CODE REVIEW

File: modules/database/variables.tf [hunk 1]
```diff
@@ -36,6 +36,18 @@ variable "database_engine" {
   description = "Which database engine to use, currently supports `postgres` or `mysql`"
 }
 
+variable "database_engine_version" {
+  description = "Which database version to use. See the aws cli describe-db-engine-versions command for a list of valid versions"
```
Comment (davidcheung) [inline modules/database/variables.tf:40]:
```suggestion
  description = "Which database version to use. See the aws rds describe-db-engine-versions command for a list of valid versions"
```

Comment (davidcheung) [reply modules/database/variables.tf:40]: should these be `aws rds describe-db-engine-versions` instead?

**Review by davidcheung — APPROVED** [review]
LGTM

File: modules/database/variables.tf (hunk 1 above)
Comment (bmonkman) [reply modules/database/variables.tf:40]: I meant to check the command in the aws cli, but yeah that works too.

**Pull request merged**